src/app/(private)/emergency-evacuation-applications/[id]/page.tsx

```diff
@@ -90,8 +91,12 @@ const Page = ({
                               {...field}
                               disabled
                               defaultValue={
-                                emergencyEvacuationApplicationDetails.referenceNumber ??
-                                ''
+                                emergencyEvacuationApplicationDetails?.referenceNumber ?
+                                  formatReferenceNumber(
+                                    emergencyEvacuationApplicationDetails.referenceNumber
+                                  )
+                                  :
```

Comment:
"emergencyEvacuationApplicationDetails?.referenceNumber" burada ? işaretine gerek var mı? sanırım yok diğer yerlerde de böyle bi durum varsa değiştirebiliriz

Reply:
Backend olarak kontrol etmedim, ama UI'ı güvenceye almak için eklemiştim.

Reply:
{!isLoading && !error && emergencyEvacuationApplicationDetails && (

yukarda böyle bir check olduğu için gerek yok diye düşündüm, sen de gerek olmadığını düşünüyosan buradan ve diğer yerlerden silebiliriz 👍

Reply:
Gerekli düzenlemeler yapılıp, kontrol edildi. Teşekkürler 🙏🏼